spring-boot-modules/spring-boot-keycloak/spring-boot-resource-server/src/main/java/com/baeldung/boot/keycloak/resourceserver/MeController.java

```diff
@@ -0,0 +1,24 @@
+package com.baeldung.boot.keycloak.resourceserver;
+
+import java.util.List;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.oauth2.core.oidc.StandardClaimNames;
+import org.springframework.security.oauth2.server.resource.authentication.JwtAuthenticationToken;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * @author ch4mp&#64;c4-soft.com
+ */
```

Comment:
Let's remove these author tags

Reply:
Why that? 

I contribute for free. All I get are the credits for authoring what I write for Baeldung. These tags are something I put, for instance, when contributing to the Spring Framework.